src/EditorFeatures/TestUtilities2/Utilities/GoToHelpers/MockSymbolNavigationServiceFactory.vb

```diff
@@ -0,0 +1,17 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.Host
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.Navigation
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Utilities.GoToHelpers
+    <ExportWorkspaceServiceFactory(GetType(ISymbolNavigationService), ServiceLayer.Editor), [Shared]>
```

Comment:
❗️ Needs the following:

1. Use `ServiceLayer.Test`
2. Add `<PartNotDiscoverable>`

Reply:
➡️ `ServiceLayer.Test` isn't a thing, so `ServiceLayer.Host` was used instead.